recipes/cloudwatch_agent_install.rb

```diff
@@ -33,8 +33,8 @@
 package_url_prefix = "#{s3_domain}/amazoncloudwatch-agent-#{node['cfncluster']['cfn_region']}"
 arch_url_component = arm_instance? ? 'arm64' : 'amd64'
 platform_url_component = value_for_platform(
-  # No CW Agent for CentOS 8 ARM, using RHEL package
-  'centos' => { '>=8.0' => arm_instance? ? 'redhat' : node['platform'] },
+  # No CW Agent for CentOS ARM, using RHEL package
+  'centos' => { '>=7.0' => arm_instance? ? 'redhat' : node['platform'] },
```

Comment:
Since we deprecated CentOS6 already, we can use `default` instead of `>=7.0` to use RHEL CW agent for all CentOS OS we currently support

Reply:
Updated. Thanks!